tests/models/test_gptq_bitblas.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
If these tests are light enough (which seems like it), let's also add it to `.buildkite`.